src/main/java/com/hubspot/jackson/datatype/protobuf/builtin/serializers/MessageSerializer.java

```diff
@@ -39,9 +41,17 @@ public MessageSerializer(ExtensionRegistryWrapper extensionRegistry) {
   }
 
   public MessageSerializer(ProtobufJacksonConfig config) {
-    super(MessageOrBuilder.class);
+    this(config, false);
+  }
+
+  public MessageSerializer(ProtobufJacksonConfig config, boolean unwrappingSerializer) {
+    this(config, null, unwrappingSerializer);
+  }
 
+  public MessageSerializer(ProtobufJacksonConfig config, NameTransformer nameTransformer, boolean unwrappingSerializer) {
```

Comment:
It doesn't look like the `NameTransformer` is used, which is probably fine? But in that case we can probably remove this constructor and just use `MessageSerializer(ProtobufJacksonConfig config, boolean unwrappingSerializer)` constructor (and make that one private)

Reply:
Thanks a ton @jhaber . Implemented your comments.